apis/types/container_json.go

```diff
@@ -43,6 +43,9 @@ type ContainerJSON struct {
 	// host config
 	HostConfig *HostConfig `json:"HostConfig,omitempty"`
 
+	// The rootfs path of the container.
+	HostRootPath string `json:"HostRootPath,omitempty"`
```

Comment:
Can we add test case to make sure `inspect` command can get the `HostRootPath` ?

Reply:
Sure

Reply:
fixed

Reply:
I am afraid that @HusterWan means that you could a test case in file `test/cli_inspect_test.go` rather than `cli/inspect_test.go`. 
Thus we could cover this feature in integration test, not only the unit test. @fengzixu